website/src/css/_config.scss

```diff
@@ -0,0 +1,348 @@
+// All of the following configs are used in the website and documentation provided by the Design team Glyph
+
+// Colors
+
+@mixin detox-colors($prefix) {
+  --#{$prefix}-r0: #f14c52ff;
+  --#{$prefix}-y0: #f7ba29ff;
+  --#{$prefix}-b0: #3e77e4ff;
+  --#{$prefix}-u0: #7633e5ff;
+  --#{$prefix}-e0: #3ebf72ff;
+
+  --#{$prefix}-r1a: #f14c5289;
+  --#{$prefix}-y1a: #f7ba2989;
+  --#{$prefix}-b1a: #3e77e489;
+  --#{$prefix}-u1a: #7633e589;
+  --#{$prefix}-e1a: #3ebf7289;
+
+  --#{$prefix}-r2a: #f14c5259;
+  --#{$prefix}-y2a: #f7ba2959;
+  --#{$prefix}-b2a: #3e77e459;
+  --#{$prefix}-u2a: #7633e559;
+  --#{$prefix}-e2a: #3ebf7259;
+
+  --#{$prefix}-r3a: #f14c5240;
+  --#{$prefix}-y3a: #f7ba2940;
+  --#{$prefix}-b3a: #3e77e440;
+  --#{$prefix}-u3a: #7633e540;
+  --#{$prefix}-e3a: #3ebf7240;
+
+  --#{$prefix}-r4a: #f14c5226;
+  --#{$prefix}-y4a: #f7ba2926;
+  --#{$prefix}-b4a: #3e77e426;
+  --#{$prefix}-u4a: #7633e526;
+  --#{$prefix}-e4a: #3ebf7226;
+
+  --#{$prefix}-r5a: #f14c5212;
+  --#{$prefix}-y5a: #f7ba2912;
+  --#{$prefix}-b5a: #3e77e412;
+  --#{$prefix}-u5a: #7633e512;
+  --#{$prefix}-e5a: #3ebf7212;
+
+  --#{$prefix}-a0: #12161bff;
```

Comment:
I would separate them to detox-colors-light.

Reply:
Good idea. Done - thanks!